Lib/idlelib/idle_test/test_help_about.py

```diff
@@ -33,6 +34,12 @@ def test_dialog_title(self):
         """Test about dialog title"""
         self.assertEqual(self.dialog.title(), 'About IDLE')
 
+    def test_dialog_logo(self):
+        """Test about dialog logo."""
+        path, file = os.path.split(self.dialog.icon_image['file'])
+        fn, ext = os.path.splitext(file)
+        self.assertEqual(fn, 'idle_48')
```

Comment:
Thanks.  I wasn't sure if I should mock the PhotoImage just to show that the widget is created.